integration_tests/keysend_test.go

```diff
@@ -128,15 +127,15 @@ func (suite *KeySendTestSuite) TestMultiKeysend() {
 		Keysends: []v2controllers.KeySendRequestBody{
 			{
 				Amount:      150,
-				Destination: "03abcdef123456789a",
+				Destination: "123456789012345678901234567890123456789012345678901234567890abcdef",
 			},
 			{
 				Amount:      100,
-				Destination: "03abcdef123456789a",
+				Destination: "123456789012345678901234567890123456789012345678901234567890abcdef",
 			},
 			{
 				Amount:      50,
-				Destination: "03abcdef123456789a",
+				Destination: "123456789012345678901234567890123456789012345678901234567890abcdef",
```

Comment:
I would add 2 more tests: 
- invalid destination pubkey hex
- Expired invoice
I added a commit that should let you set the expiry when calling `mlnd.AddInvoice`.